themes/custom_theme/valencia-theme/src/css/_custom.scss

```diff
@@ -217,6 +212,16 @@ ul {
 		&:hover {
 			color: $accent-color;
 		}
+		p {
```

Comment:
We should use CSS classes as much as possible rather than relying on HTML elements that might or might not exist on the page.